coalib/results/result_actions/IgnoreResultAction.py

```diff
@@ -65,3 +67,29 @@ def apply(self, result, original_file_dict, file_diff_dict, language: str,
             file.writelines(ignore_diff.modified)
 
         return file_diff_dict
+
+    def get_ignore_comment(self, origin, language):
+        try:
+            comment_delimiter = Language[
+                language].get_default_version().comment_delimiter
+            ignore_comment = (str(comment_delimiter) + ' Ignore ' +
+                              origin + '\n')
+        except AttributeError:
+            # singleline comments not supported by language
+            try:
+                multiline_comment_delimiter = Language[
+                    language].get_default_version().multiline_comment_delimiters
+                start_comment, end_comment = list(
+                                        multiline_comment_delimiter.items())[0]
```

Comment:
in that case rather use `start_comment, end_comment = next(iter(multiline_comment_delimiter.items())))` for performance, your version would iterate over the whole dict just to get the first element